examples/swaps/auraBalSwap.ts

```diff
@@ -0,0 +1,139 @@
+/**
+ * Example showing how to swap auraBal which has its own setup via AuraBalSwap.
+ * Note - this is only supported on Balancer V2 and on mainnet.
+ * (Runs against a local Anvil fork)
+ *
+ * Run with:
+ * pnpm example ./examples/swaps/auraBalSwap.ts
+ */
+import { createTestClient, http, publicActions, walletActions } from 'viem';
+import {
+    BALANCER_RELAYER,
+    ChainId,
+    CHAINS,
+    Relayer,
+    Slippage,
+    AuraBalSwap,
+    Token,
+    TokenAmount,
+    SwapKind,
+} from '../../src';
+import { ANVIL_NETWORKS, startFork } from '../../test/anvil/anvil-global-setup';
+import { makeForkTx } from 'examples/lib/makeForkTx';
+import { getSlot } from 'examples/lib/getSlot';
+import { auraBalToken } from '@/entities/swap/swaps/v2/auraBalSwaps/constants';
+import { exit } from 'process';
+
+const auraBalSwap = async ({ rpcUrl, client, userAccount, chainId }) => {
+    // Create AuraBalSwap instance which can be used to query & buildCalls
+    const auraBalSwap = new AuraBalSwap(rpcUrl as string);
+
+    const tokenIn = new Token(
+        1,
+        '0xba100000625a3754423978a60c9317c58a424e3D', // BAL
+        18,
+    );
+    const tokenOut = auraBalToken;
```

Comment:
Can you help me better understand context/audience for this example script? I'm curious why import `auraBalToken` from constants for `tokenOut` but show the explicit creation for `tokenIn`. 


Explicit addresses and creation all seem to be shown in examples for v3 SDK docs

Reply:
It was mainly to serve as an example to FE team so they can see how to use it. I originally thought using `auraBalToken` was helpful but I think you're right and its better to keep the same standard.